### DISCUSSION TITLE
Does Survey.jl Subsume Methods from `pewmethods`?

### DISCUSSION THREAD

**Discussion by TheCedarPrince** [xKDR/Survey.jl#323] [General]
Came across this [helpful R package](https://github.com/pewresearch/pewmethods) that builds around `survey` (the R package). Does Survey.jl support the methods enumerated in `pewmethods` for the most part?

Just was a bit of a vague question but was curious. Thanks!

**Comment by arnavk23** [comment]
Hi @TheCedarPrince — great question!

Survey.jl shares a similar design philosophy to R's survey package and is indeed capable of many of the core operations found in pewmethods, especially those involving weighted means, totals, proportions, and subgroup estimates.

Direct support: Basic estimators and variance computations are already implemented in Survey.jl, and many functions in pewmethods (which wraps and simplifies survey) map closely to these.

Not yet subsumed: Higher-level helpers in pewmethods (like automated crosstabs with MOE, or framed summary tables) are not yet fully replicated in Survey.jl. But the underlying mechanics to compute them are available.

Design direction: Provide lower-level primitives for design-based inference, and tools like pewmethods could eventually be layered on top in Julia or ported in a similar form.

**Comment by ayushpatnaikgit** [comment]
Indeed the Prof Lumley's package is amazing! 

We built the Julia package when we found the R package to be slow.
In the same amount of time, you can do at least 10x more bootstrap samples in Julia.
